### ISSUE TITLE
eachRow does not accept options

### ISSUE THREAD

**Issue by rashtao** [azuqua/cassanknex#8]
CQL driver supports an options parameter (http://docs.datastax.com/en/developer/nodejs-driver/2.1/nodejs-driver/reference/paging.html) that is foundamental for:
- setting the autopaging: { autoPage : true }
- specify the page to fetch: { pageState : pageState, prepare :  1 , fetchSize :  1000 }

Currently cassanknex only allows me only to fetch the first 5000 rows and then I don't know how to get the next rows.

It would be great to have the possibility to specify the options, maybe following the fluent interface approach: 

```js
cassanKnex("mykeyspace")
            .select()
            .from("users")
            .options({prepare: true, autoPage : true})
            .eachRow();
```

**Comment by rashtao** [comment]
The 'options' method would also be used for regular queries:
```js
                cassanKnex("mykeyspace")
                    .insert(obj)
                    .into("users")
                    .options({prepare: true})
                    .exec(cb);
```

**Comment by UnbounDev** [comment]
Currently options can be passed to `exec` and `stream` by calling the methods w/ an `options` object as the first parameter in the call signature (the docs are not very clear on this); e.g.

```
cassanKnex("mykeyspace")
                    .insert(obj)
                    .into("users")
                    .options({prepare: true})
                    .exec({ pageState : pageState, prepare : 1 , fetchSize : 1000 }, cb);
```

However; this is not implemented for `forEach` (my oversight); I'll add the same functionality to `eachRow` s.t. you can pass options via `forEach(options, rowCb, errorCB)`.

**Comment by UnbounDev** [comment]
@rashtao the fix is in! Version 1.6.0 on npm, [docs](https://github.com/azuqua/cassanknex#QueryExecutors) have been updated as well - thanks for bringing attention to the issue!
